src/Partials/RoundedColorButton.vala

```diff
@@ -0,0 +1,53 @@
+/*
+* Copyright (c) 2021 Alecaddd (http://alecaddd.com)
+*
+* This file is part of Akira.
+*
+* Akira is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 3 of the License, or
+* (at your option) any later version.
+
+* Akira is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU General Public License for more details.
+
+* You should have received a copy of the GNU General Public License
+* along with Akira.  If not, see <https://www.gnu.org/licenses/>.
+*
+* Authored by: Abdallah "Abdallah-Moh" Mohammad <abdullah_mam1@icloud.com>
+*/
+
+public class Akira.Partials.RoundedColorButton : Gtk.Button {
+    public string background_color;
+
+    public RoundedColorButton (string bg_color) {
+        background_color = bg_color;
+        can_focus = false;
+        width_request = 25;
+        height_request = 25;
```

Comment:
You can put this inline: `width_request = height_request = 24;`
Let's use 24 so we can better round em up.